src/components/TheNav.vue

```diff
@@ -44,24 +41,22 @@
                         class="absolute top-0 left-0 right-0 h-screen bg-dark-yellow text-2xl"
                     >
                         <ul class="nav-items">
-                            <li
-                                @click="toggleNav(item.callback)"
-                                v-for="item in navItems"
-                                :key="item"
-                                v-show="isSignedIn === item.conditionOne
-                                || isSignedIn === item.conditionTwo"
-                            >
-                                <router-link
-                                    :to="item.path"
-                                    :class="{ active: $route.name === item.activeCondition }"
-                                    class="px-6 py-2">
-                                    {{ item.content }}
-                                </router-link>
-                            </li>
+                          <li
+                            @click="toggleNav(callback)"
```

Comment:
this is wrong. What's the first parameter on an event handler? Do you know why and how to fix it?

Reply:
I guess that would be the event itself? But it's my understanding I don't need to put it as a argument if I don't actually use it. So, are you suggesting it should be like that: `@cllick="toggleNav(e, callback)"`? Won't is cause any issues?

Reply:
It could. Do you know how we could get rid of event being the first param there?

Reply:
No idea, I tried to quickly find the answer but I haven't managed to do that 😪

Reply:
Two ways would do:
1. Wrap (_encapsulate_) `toggleNav` with another function
2. Check if passed argument (`cb`) is an instance of an event